graylog2-web-interface/test/helpers/mocking/index.js

```diff
@@ -1,4 +1,7 @@
 export { default as CombinedProviderMock } from './CombinedProviderMock';
+export { default as MockCombinedProvider } from './CombinedProviderMock';
 export { default as StoreMock } from './StoreMock';
+export { default as MockStore } from './StoreMock';
 export { default as StoreProviderMock } from './StoreProviderMock';
+export { default as MockStoreProvider } from './StoreProviderMock';
 export { default as asMock } from './AsMock';
```

Comment:
I guess it is a possible improvement for a follow up PR to replace the usage of `StoreProviderMock` and `StoreMock` in all our tests?

Reply:
Yes, that was the plan. Good catch!